catalyst-gateway/Earthfile

```diff
@@ -84,9 +84,16 @@ all-hosts-build:
 # package : Create a deployable container for catalyst-gateway
 package:
     FROM debian:12.7-slim
-    WORKDIR /cat-gateway
+
+    WORKDIR /app
+    RUN apt-get update && apt-get install -y curl python3 python3-pip
+    RUN pip3 install --break-system-packages cqlsh
+    RUN curl https://certs.secureserver.net/repository/sf-class2-root.crt -o /tmp/keyspaces.crt
```

Comment:
I don't think we need this cert anymore?

Reply:
We don't. We can remove that and clean up the volumes and mounts associated with it.